scripts/build.py

```diff
@@ -1101,6 +1125,23 @@ def generate_build_directory():
     print 'Build completed.'
 
 
+def compile_typescript_files(
+        file, outDir='backend_prod_files', allowJS='true',
+        lib='es2017,dom', noImplicitUseStrict='true', rootDir='.',
+        skipLibCheck='true', target='es5',
+        typeRoots='../node_modules/@types'):
+
+    print 'Compiling %s...' %file
```

Comment:
`file` is a [keyword](https://docs.python.org/2/library/functions.html#file), maybe pick a different name (filepath?)

Reply:
Done.